### ISSUE TITLE
Methods such as ``getSequences()`` and ``graph_analysis()`` is not defined in currently released code

### ISSUE THREAD

**Issue by bjchan9an** [qian-feng/Gencoding#3]
We find it difficult to reproduce this work since currently released code is not complete.
Methods such as ``getSequences()`` and ``graph_analysis()`` is not defined in currently released code.
Hope you can fill in the missing code ;)

**Comment by GentleCP** [comment]
Yep, except that, this project is really terrible since there is no any comment in the program. Maybe the author is too lazy to write them, but it makes users quite hard to understand and reimplement the demo.

**Comment by hexbo** [comment]
Hi @bjchan9an and @GentleCP, Did you succeed in reproducing this job? I encountered some problems in reproducing ACFG feature extraction. I would be grateful if you could give me some advice. Thanks very much.

**Comment by YuanBoXie** [comment]
I finally finished these problems. The getSequences() and graph_analysis() both are ida-python built-in functions. 
Due to the time factor, the current code needs to be modified. It is better to translate it into the python 3 version, which can be executed in IDA Pro.
I also encountered problems when saving data,  so it's necessary to write an Encoder for data JSON serialization later.

**Comment by yizhicheng1** [comment]
Can you please send me your modified code, I also encountered some problems

**Comment by yizhicheng1** [comment]
Can you please send me your modified code, I also encountered a lot of problems. Thank you so much.

**Comment by 20521862** [comment]
@hexbo Can you give me contact information? I need your help with some information regarding converting IDA pro to JSONL
20521862@gm.uit.edu.vn
